include/rocksdb/db.h

```diff
@@ -501,10 +501,7 @@ class DB {
   virtual Status Merge(const WriteOptions& /*options*/,
                        ColumnFamilyHandle* /*column_family*/,
                        const Slice& /*key*/, const Slice& /*ts*/,
-                       const Slice& /*value*/) {
-    return Status::NotSupported(
-        "Merge does not support user-defined timestamp yet");
-  }
+                       const Slice& /*value*/) = 0;
```

Comment:
I don't think this has to be pure virtual; we do have an implementation in `db_impl_write.cc`, right?